rocketmq-storm/README.md

```diff
@@ -0,0 +1,2 @@
+# RocketMQ Storm Integration
```

Comment:
No, this is just a placeholder for rocketmq-storm new `README.md` file. I plan to add some description in this file. It may be better to do this in another PR.